src/services/ghost/GhostStrategy.ts

```diff
@@ -283,6 +284,121 @@ ${sections.filter(Boolean).join("\n\n")}
 `
 	}
 
+	private safeJsonParse(jsonContent: string): any {
+		try {
+			// First, try direct parsing
+			return JSON.parse(jsonContent)
+		} catch (error) {
+			// If direct parsing fails, try to fix common issues with escaped content
```

Comment:
Have you checked if there's an existing function that does this? I think there might be.